src/App.css

```diff
@@ -13,21 +13,23 @@ body {
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/234454982-a2f7b3dc-d267-45a7-ab53-27637ff3a801.png)

**[OPTIOANAL]**

- [ ] Kindly remove this live demo section since you don't have any live link for your application :thinking: Once you would have a live link then you can add this live demo section again in your readme to make your readme professional and more useful :+1: